client/src/components/CustomDateInput.js

```diff
@@ -95,7 +95,8 @@ CustomDateInput.propTypes = {
   value: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.number,
-    PropTypes.instanceOf(Date)
+    PropTypes.instanceOf(Date),
+    PropTypes.oneOf([null])
```

Comment:
Since the prop is not required, are you sure this `null` type is needed?

Reply:
Actually I just wanted to add it to clarify that from now on our date can be nullable. I thought that when we check it later in the future it would be helpful knowing that it is okay not to have date. I can remove it if it is not needed.

Reply:
That's what `PropTypes.isRequired` is there for, to indicate that a (non-null and non-undefined) value *must* be passed.

Reply:
Alright, I'll remove it then.